contributing/code_of_conduct/index.rst

```diff
@@ -0,0 +1,91 @@
+Code of Conduct
+===============
+
+Our Pledge
+----------
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnic origin, gender identity and expression, level of experience,
+education, socio-economic status, nationality, personal appearance,
+religion, or sexual identity and orientation.
+
+Our Standards
+-------------
+
+Examples of behavior that contributes to creating a positive environment
+include:
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or
+  advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic
+  address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+
+Our Responsibilities
+--------------------
+
+:doc:`Enforcement team members </contributing/code_of_conduct/enforcement_team>`
+are responsible for clarifying the standards of acceptable
+behavior and are expected to take appropriate and fair corrective action in
+response to any instances of unacceptable behavior.
+
+Enforcement team members have the right and responsibility to remove, edit, or
+reject comments, commits, code, wiki edits, issues, and other contributions
+that are not aligned to this Code of Conduct, or to ban temporarily or
+permanently any contributor for other behaviors that they deem inappropriate,
```

Comment:
I can't remember it was asked, at least not directly. As I understand it's possible to ban a contributor only on the managed platforms, i.e. github (https://help.github.com/articles/blocking-a-user-from-your-organization/), slack and so on. A contribution is mentioned above as `comments, commits, code, wiki edits, issues, and other contributions`. While the `other contributions` is open for interpretation (e.g. a slack conversation).

Reply:
I would see slack included here .. I am hestitant to explicitly mention all our “official” platforms of communication, since it means we need to update this frequently. but if it helps clarity maybe we should?

so what are the official places?

* github
* symfony.com
* slack
* symfony events

That being said, I am also hoping that some projects within our eco-system start adopting our CoC and enforcement process. So in that respect explicit listing might not be ideal. I was hoping for us to implement GDPR which would have entailed actually asking people to consent to the CoC on all our platforms, which in turn would mean that if anyone also adopts our CoC, they would then ask for consent on those additional platforms.